src/middlewares/error-middleware.js

```diff
@@ -1,9 +1,11 @@
 import ApiError from '#exceptions/api-error.js'
 
-export default function (err, req, res, next) {
+export default function errorMiddleware(err, req, res, next) {
   if (err instanceof ApiError) {
+    next()
```

Comment:
The next() call before returning the response is redundant. The return res.status(...) statement will end the function execution, so the next() calls after that are not needed